e2e/cypress/tests/integration/files_and_attachments/disabled_file_upload_spec.js

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @files_and_attachments
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Upload Files - Settings', () => {
+    let channelUrl;
+
+    before(() => {
+        // # Go to system admin then verify admin console URL and header
```

Comment:
```suggestion
// Group: @not_cloud @files_and_attachments

import * as TIMEOUTS from '../../fixtures/timeouts';

describe('Upload Files - Settings', () => {
    let channelUrl;

    before(() => {
        cy.shouldNotRunOnCloudEdition();

        // # Go to system admin then verify admin console URL and header
```

Reply:
I just checked and found that file sharing is restricted to be changed in cloud. With that, please add flags so this could be filtered out in cloud test.

Reply:
Thanks @saturninoabril I just commited your suggestion